BrowserKit/Sources/ComponentLibrary/Buttons/LinkButton.swift

```diff
@@ -25,8 +25,7 @@ open class LinkButton: UIButton, ThemeApplicable {
         updatedConfiguration.title = viewModel.title
         updatedConfiguration.titleTextAttributesTransformer = UIConfigurationTextAttributesTransformer { incoming in
             var outgoing = incoming
-            outgoing.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
-                                                                   size: viewModel.fontSize)
+            outgoing.font = FXFontStyles.Regular.body.scaledFont().withSize(viewModel.fontSize)
```

Comment:
thanks @bmihai23! tagging @cwzilla to confirm if link button texts should respect the size of design system or if we want to keep it static.

Reply:
@bmihai23 Since we want to pull from the design system now and avoid hardcoding the font size, could we replace `viewModel.fontSize` with a new property `viewModel.font` of type `FXFontStyles`? Therefore, we initialize the viewModel with a `FXFontStyles` type in the areas we are using the button and we can do `outgoing.font = viewModel.font` here. 

It's a bit more than the ask of the original ticket so let me know what you think and if this is something you want to tackle on!

Reply:
Hi @cyndichin, thanks for bringing this up! Yes, I'm happy to take on the task 🙏

Reply:
awesome thank you @bmihai23! Will give another look when its ready :)